connectivity/cellular/source/framework/RIL/RIL_CellularContext.cpp

```diff
@@ -0,0 +1,1164 @@
+/*
+ * Copyright (c) 2020, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "RIL_CellularContext.h"
+#include "RIL_CellularDevice.h"
+#include "RIL_CellularNetwork.h"
+#include "RILAdaptation.h"
+#include "LWIPStack.h"
+#include "CellularLog.h"
+#include "ScopedLock.h"
+#include "mbed.h"
```

Comment:
Including `mbed.h` is allowed only for application code, so replace these (there are multiple these in this PR) with the required headers.

Reply:
This issue has been modified during the resolution of Travis failures, and the code will be submitted later.